data/category-cryptocurrency

```diff
@@ -16,23 +16,31 @@ include:zb
 bitbank.cc
 bitcoin.org
 bitfinex.com
+bitget.com
 bitmex.com
 bitquick.co
 bitstamp.net
 bittrex.com
 btcbox.co.jp
 cex.io
 coinbase.com
+coingecko.com
 coinglass.com
+coinmarketcap.com
+crypto.com
 cryptocompare.com
 curve.fi
 debank.com
 dogecoin.com
+gate.io
 infura.io
+mexc.com
+mexcsensors.com
```

Comment:
Could you provide any evidence to prove this domain is belonging to this category?